resize-observer/devicepixel2-ref.html

```diff
@@ -0,0 +1,41 @@
+<!doctype html>
+<body>
+<script>
+const patternSize = 4;
+
+function createPatternDataURL() {
+  const ctx = document.createElement('canvas').getContext('2d');
+  ctx.canvas.width = patternSize;
+  ctx.canvas.height = patternSize;
+
+  const b = [0, 0, 0, 255];
+  const t = [0, 0, 0, 0];
+  const r = [255, 0, 0, 255];
+  const g = [0, 255, 0, 255];
+
+  const imageData = new ImageData(4, 4);
+  imageData.data.set([
+    b, t, r, t,
+    t, b, t, g,
+    r, t, b, t,
+    t, g, t, b,
```

Comment:
You've got a different pixel-pattern here vs. in the testcase, I think by accident? This makes the test fail in all browsers for un-interesting reasons right now. :)  (Though it's hard to see without e.g. using an accessibility tool to scale up the device pixels.)

If I fix this (e.g. using the testcase's setup in the reference case), then I confirmed that the test passes for me in the WPT harness (including with the other changes I suggested -- using `-x -y` in device pixels and removing the `#front` css rule)

Reply:
done